### PR TITLE
improved design & system commands docs : Meshmodel guide

### PR DESCRIPTION
**Pull request by parth721**
**Notes for Reviewers**

This PR fixes #

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit 930efdb8038b89999750b5d142b7fb97298dc1cd at: https://64c54ee603fc330d22a9af1b--meshery-docs-preview.netlify.app

**Comment by sudhanshutech** [comment]
Let's discuss it on the websites call.
Please add this as an agenda item in the meeting minutes, if you would :)

https://docs.google.com/document/d/1XczAHXVe2FIWPqiF57ospJ43zw5cZQ7ui8mn39v5EvA/edit#heading=h.lohhtewfwima

**Comment by l5io** [comment]
🚀 Preview for commit 14088c2d1bda2ff1d503625a0940852509c8374f at: https://64c65a28b41ae17f7dade965--meshery-docs-preview.netlify.app

File: docs/pages/guides/mesheryctl/mesheryctl_system.md [hunk 1]
```diff
@@ -0,0 +1,124 @@
+---
```
Comment (leecalcote) [inline docs/pages/guides/mesheryctl/mesheryctl_system.md:1]: @alphaX86 @zakisk @abdullah1308 will this page be overwritten by Mesheryctl's self-publishing workflow? 
https://docs.meshery.io/reference/mesheryctl/system

Comment (suhail34) [reply docs/pages/guides/mesheryctl/mesheryctl_system.md:1]: I think the guides doesn't comes under self-publishing workflow it is only for [mesheryctl command reference](https://docs.meshery.io/reference/mesheryctl/system)

**Pull request closed without merging**